packages/orbit-components/src/Wizard/Wizard.stories.tsx

```diff
@@ -113,3 +118,43 @@ export const Playground = () => {
 Playground.story = {
   parameters: { knobs: { escapeHTML: false } },
```

Comment:
I am stuck on this one, controls doesn't have this param yet - it was suggested to be added but the issue is still open after 4 years. Any suggestions how to approach this?

Reply:
There should be no difference, removing then 👍